code/obj/machinery/door/airlock.dm

```diff
@@ -1594,6 +1594,22 @@ About the new airlock wires panel:
 		return src.Attackhand(user)
 	else if (ispryingtool(C))
 		src.unpowered_open_close()
+	else if (istype(C, /obj/item/sword) && (user.a_intent == INTENT_HELP) && !src.operating)
```

Comment:
```suggestion
	else if ((C.type == /obj/item/sword) && (user.a_intent == INTENT_HELP) && !src.operating)
```
Unless you want the discount variant to also melt through doors, but that would mess with the melt vs bash timings since it's a lot weaker. (yes it sucks that csabers are just `/obj/item/sword`)